homeassistant/components/gios/sensor.py

```diff
@@ -49,66 +46,142 @@
 
 
 @dataclass
-class GiosSensorEntityDescription(SensorEntityDescription):
-    """Class describing GIOS sensor entities."""
+class GiosSensorRequiredKeysMixin:
+    """Class for GIOS entity required keys."""
+
+    value: Callable[[GiosSensors], StateType]
+
+
+@dataclass
+class GiosSensorEntityDescription(SensorEntityDescription, GiosSensorRequiredKeysMixin):
+    """GIOS sensor entity description."""
 
-    value: Callable | None = round
+    subkey: str | None = None
 
 
 SENSOR_TYPES: tuple[GiosSensorEntityDescription, ...] = (
     GiosSensorEntityDescription(
         key=ATTR_AQI,
         name="AQI",
-        value=None,
+        value=lambda sensors: getattr(sensors.aqi, ATTR_VALUE, None),
```

Comment:
We need a default value for `getattr` because mypy complains that `Any` type is returned instead of `Union[str, int, float, None]`.